nginx/nginx.conf

```diff
@@ -22,3 +22,56 @@ server {
         }
     }
 }
+
+server {
+    listen       15001;
+    charset UTF-8;
+
+    location / {
+        proxy_pass http://read-model:15000/;
+
+        # Handle CORS headers
+        proxy_set_header Host $host;
```

Comment:
Should we use instead of duplication - single defined CORS.conf as well as charset?
https://www.juannicolas.eu/how-to-set-up-nginx-cors-multiple-origins/
![image](https://github.com/user-attachments/assets/97d9ed45-8c59-4a66-a138-f14e61407364)